build/check-images.ts

```diff
@@ -144,9 +145,7 @@ export function checkImageReferences(
       // insensitively.
 
       // What follows uses the same algorithm as FileAttachment.findByURLWithFallback
-      // but only adds a filePath if it exists for the DEFAULT_LOCALE
-      const filePath = FileAttachment.findByURL(finalSrc);
-      let enUSFallback = false;
+      let filePath = FileAttachment.findByURL(finalSrc);
```

Comment:
@yin1999 Could we replace lines 148 to 160 with `findByURLWithFallback` now that we don't need `enUSFallback` anymore?

```suggestion
      let filePath = FileAttachment.findByURLWithFallback(finalSrc);
```